### DISCUSSION TITLE
Feature Request: Please add a log off button

### DISCUSSION THREAD

**Discussion by Mashiane** [pockethost/pockethost#147] [Q&A]
Hi

For some reason, at times when I open pockethost on the browser, it does not show the buttons for showing the dashboard or projects.

Its is possible to add a "Sign Out" button so that one can click that to clear any running sessions and login again.

I'm not really sure what could be the cause, but perhaps this could help if its constant to just restart a previously signed in sessio.

Thank you so much

**Comment by benallfree** [comment]
@Mashiane You mean this?

<img width="471" alt="image" src="https://user-images.githubusercontent.com/1068356/209542881-eebc44f5-787a-4542-a312-35674ee6e816.png">
